### PR TITLE
Add subtitle field for Post model

### PR DESCRIPTION
**Pull request by protoroto**
This add a `subtitle` field on the `Post` model, and it is shown in templates only if filled with something.
Related to #444

### CODE REVIEW

File: djangocms_blog/models.py [hunk 1]
```diff
@@ -228,6 +228,7 @@ class Post(KnockerModel, BlogMetaMixin, TranslatableModel):
         title=models.CharField(_('title'), max_length=767),
         slug=AutoSlugField(_('slug'), max_length=767, blank=True,
                            db_index=True, allow_unicode=True),
+        subtitle=models.TextField(verbose_name=_('subtitle'), blank=True, default=''),
```
Comment (yakky) [inline djangocms_blog/models.py]: I think we should use a charfield here

Comment (protoroto) [reply djangocms_blog/models.py]: Sure! Any idea about the max_length? Was thinking at 767 as the title, but maybe we could need more.

Comment (yakky) [reply djangocms_blog/models.py]: 767 is a maximum due to mysql fields constraint

Comment (protoroto) [reply djangocms_blog/models.py]: :+1:

**Pull request merged**